lib/src/presentations/applicant_profile/screen/applicant_profile_view.dart

```diff
@@ -0,0 +1,263 @@
+                  centerTitle: true,
+                  automaticallyImplyLeading: false,
+                  expandedHeight: 0,
+                  title: _buildTabBar(onTap: tabsRouter.setActiveIndex),
+                ),
+              ],
+              body: child,
+            ),
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _buildAppBar() {
+    double width = MediaQuery.sizeOf(context).width;
+    return BlocBuilder<ApplicantProfileCubit, ApplicantProfileState>(
+      builder: (context, state) {
+        return SliverAppBar(
+          actions: [
+            IconButton(
+              onPressed: () {},
+              icon: SvgPicture.asset(AppImages.setting),
+            ),
+            const SizedBox(width: 5),
+          ],
+          leading: IconButton(
+            onPressed: () => context.router.pop(),
+            icon: const Icon(Icons.arrow_back_rounded, color: Colors.white),
+          ),
+          backgroundColor: state.isTop ? AppColors.primary : Colors.transparent,
+          elevation: 0,
+          pinned: true,
+          centerTitle: true,
+          stretch: true,
+          expandedHeight: 0.6 * width,
+          title: AnimatedOpacity(
+            opacity: !state.isTop ? 0.0 : 1.0,
+            duration: const Duration(milliseconds: 300),
+            child: Text(
+              PrefsUtils.getUserInfo()?.name ?? "",
+              style: AppStyles.normalTextWhite
+                  .copyWith(fontWeight: FontWeight.w500),
+            ),
+          ),
+          flexibleSpace: FlexibleSpaceBar(
+            stretchModes: const [StretchMode.zoomBackground],
+            expandedTitleScale: 1.0,
+            centerTitle: false,
+            titlePadding: const EdgeInsets.all(16),
+            background: _buildBackgroundAppbar(),
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _buildBackgroundAppbar() {
+    double width = MediaQuery.sizeOf(context).width;
+    return SizedBox(
+      width: width,
+      height: 0.6 * width,
+      child: SafeArea(
+        child: Stack(
+          children: [
+            Container(
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(36),
+                gradient: const LinearGradient(
+                  colors: [Color(0xFF130160), Color(0xFF36353C)],
+                  transform: GradientRotation(0.2),
+                ),
+              ),
+              width: width,
+              height: 0.6 * width,
+              child: SvgPicture.asset(AppImages.profileBackground),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(AppDimens.smallPadding),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  const SizedBox(height: 40),
+                  ClipOval(
+                    child: CachedNetworkImage(
+                      imageUrl: PrefsUtils.getUserInfo()!.avatar,
```

Comment:
You should separate and name a variable for this line. And must check null for this case

Reply:
I fixed it